commands/highlight.js

```diff
@@ -33,21 +33,36 @@ module.exports.execute = async (client, message, args) => {
 		if(cmd === 'add') {
 
 			// If highlight is already added, say so
+			let exists = true;
+			await Highlights.count({
+				where: {
+					phrase: keywords
+				}
+			}).then(count => {
+				if(count != 0){
+					user.send('Attempted to add **`' + keywords + '`** to your highlights, but it\'s already there!');
```

Comment:
Not a change per say, but did you know that you can use backticks instead of ' ?

That allows you to reformat it like

```suggestion
					user.send(`Attempted to add \`**{keywords}**\` to your highlights, but it's already there!`);
```

Reply:
I did, but I wanted to use the backticks in the message itself to format it `like this with markdown` and I'm just used to concatenation.

I can switch it up though if it's more conventional